protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -986,14 +986,17 @@ func renderServices(services []*descriptor.Service, paths openapiPathsObject, re
 								}
 							}
 							if len(bodyExcludedFields) != 0 {
-								schema = renderMessageAsDefinition(meth.RequestType, reg, customRefs, bodyExcludedFields)
+								msgDef := renderMessageAsDefinition(meth.RequestType, reg, customRefs, bodyExcludedFields)
+								if msgDef.Properties == nil || len(*msgDef.Properties) == 0 {
+									return fmt.Errorf("not possible to create a schema for object with 0 properties for message: %s", *meth.RequestType)
+								}
```

Comment:
Thanks for this change, though I just had another thought about this. I would see this as a bug fix, but it's also probably a breaking change for users, so maybe we should just log a warning instead. See `glog.Errorf` uses in this file.